src/app/shared/services/project/organization.service.ts

```diff
@@ -59,8 +59,9 @@ export class OrganizationService {
         });
     }
 
-    getMembersForOrganization(orgID: string) {
-        return this.http.get<OrganizationMembersResponse>(`/api/project/public/organization/${orgID}/members`);
+    getMembersForOrganization(orgID: string, options = { isPublic: true }) {
+        return options.isPublic ? this.http.get<OrganizationMembersResponse>(`/api/project/public/organization/${orgID}/members`) :
+            this.http.get<OrganizationMembersResponse>(`/api/project/organization/${orgID}/members`);
```

Comment:
@mdrusko1, this was the key to why we couldn't remove members from an organization. The main difference is that the public route doesn't have UUID and email of the users, so in case when you own the organization, we need to use a private route we've used before.